packages/builder/src/pages/builder/portal/users/groups/_components/CreateEditGroupModal.svelte

```diff
@@ -11,6 +11,8 @@
   export let group
   export let saveGroup
 
+  let readonlyTitle = group?.scimInfo?.isSync
```

Comment:
Not sure on this one. The logic of this being readonly or not is part of this component. Otherwise, every single usage of this component will need to be aware of it and sent it correctly.
And the input is disabled, but the title is readonly. DisabledTitle will only make sense if you know it's sent to an input